src/Model/Serializer/ValueObject/MediaObject/VideoNormalizer.php

```diff
@@ -67,11 +32,11 @@ public function normalize($video, $format = null, array $context = []): array
             throw new InvalidArgumentException('Expected video object, got ' . get_class($video));
         }
 
-        $platformData = $this->getPlatformData($video->getUrl());
+        $platformData = VideoPlatformData::fromVideo($video);
```

Comment:
Ben persoonlijk geen fan van zulke static calls, kunnen we hier geen gewone dependency van maken zodat het makkelijker wordt om te testen?

Reply:
What to do you mean with `gewone dependency`? You want to create an interface and inject that interface inside the `VideoNormalizer`?

Reply:
Yes

Reply:
Does not need to be an extra interface, just the object itself as a dependency is fine

Reply:
But then it would be some kind of `VideoPlatformDataFactory` which again has a static method `createFromVideo`, so that is not a solution for the static method.